readme.md

```diff
@@ -6,8 +6,8 @@
 - [Electronic Computing](#electronic-computing)
 - [Boolean Logic](#boolean-logic)
 - [Number Systems](#number-systems)
-- [Registers and RAM](#registers-and-ram)
```

Comment:
TOC should have a section on "Registers cache and ram". Add "[Registers, Cache and RAM](#registers-CACHE-and-ram)" in it.